motioneye/utils/__init__.py

```diff
@@ -279,7 +279,7 @@ def parse_cookies(cookies_headers):
 
 
 def build_basic_header(username, password):
-    return 'Basic ' + base64.encodebytes(f'{username}:{password}').replace('\n', '')
+    return 'Basic %s' % base64.b64encode(f'{username}:{password}'.encode()).decode()
```

Comment:
I ask myself which variant we should prefer why (in such case where an f-string does not work):
```py
'Basic ' + ...
```
vs
```py
'Basic %s' % ...
```
The first is a a little shorter, but are there benefits using %-formatting?

Reply:
%-formatting let you concat other values than strings too. %d %s %f and so on. Then of course there is that format alternative, hih. 

```
"{price:.2f} eur".format(price = 10)) 
"{fname} age {age}".format(fname = "Smith", age = 18)
"{0} age {1}".format("Smith", 18)
"{} age {}".format("Smith", 10) 
```

I usually use %(typing format takes too much time :)) but it depends, sometimes it clearer to use format and of course f-strings are nice too. I'm usually using older python which doesn't support f-strings so please forgive me if I accidently put % instead of f-string :)

Reply:
In this case it's strings, so both works, and both fine with me 🙂.